hibernate-core/src/main/java/org/hibernate/engine/spi/SharedSessionContractImplementor.java

```diff
@@ -180,6 +181,26 @@ default long getTimestamp() {
 	 */
 	boolean isTransactionInProgress();
 
+	/**
+	 * Check if an active Transaction is necessary for the update operation to be executed.
+	 * If an active Transaction is necessary but it is not then a TransactionRequiredException is raised.
+	 *
+	 * @param exceptionMessage, the message to use for the TransactionRequiredException
+	 * @param convertException, should a ExceptionConverter be used for the TransactionRequiredException
+	 */
+	default void checkTransactionNeededForUpdateOperation(String exceptionMessage, boolean convertException) {
```

Comment:
I don't think it should be necessary to have the ```convertException``` argument.

Please see https://github.com/hibernate/hibernate-orm/pull/2780/files#r258717383